packages/styles/src/components/form-check.scss

```diff
@@ -77,6 +77,13 @@
       &:indeterminate::after {
         @include icons-mx.icon(2039);
       }
+
+      &:checked,
+      &:indeterminate {
+        @include utility-mx.high-contrast-mode {
+          background-color: SelectedItem;
```

Comment:
![image](https://github.com/swisspost/design-system/assets/1659006/aa6c68f4-5afd-44b7-a5b1-ee73d8e26b2a)

Contrast of the check mark gets really low when setting the background color. I think I'd prefer just setting the border color here (or updating the color of the checkmark).

Reply:
Yes, and it also hide for radio the selected state. Here is a fix.

Reply:
Checkboxes are good now, the only thing remaining would be the checkbox card, where I can't see the check mark:
![image](https://github.com/swisspost/design-system/assets/1659006/dbe28ddc-82ee-4cd1-83ba-229f6e4a3ed6)

Reply:
Good catch. Those card control uses more class to define background and color, so I increased specificity for HCM style.